dev/com.ibm.ws.microprofile.faulttolerance.cdi/src/com/ibm/ws/microprofile/faulttolerance/cdi/AggregatedFTPolicy.java

```diff
@@ -129,14 +111,22 @@ public FallbackPolicy getFallbackPolicy() {
     }
 
     /**
+     * @param rcInstance an instance of the request context controller
      * @return
      */
     @SuppressWarnings("unchecked")
-    public Executor<Object> getExecutor() {
+    public Executor<Object> getExecutor(Instance<AsyncRequestContextController> rcInstance) {
```

Comment:
I think it would make more sense if this was set via a setter method like every other field on the class, rather than passed in the `getExecutor` call.